contract/src/objectTools.js

```diff
@@ -3,15 +3,23 @@ import { E } from '@endo/far';
 
 const { entries, fromEntries } = Object;
 
-/** @type { <T extends Record<string, ERef<any>>>(obj: T) => Promise<{ [K in keyof T]: Awaited<T[K]>}> } */
+/**
```

Comment:
worth noting these are copied from `@agoric/internal`? someone may wonder why not import

Reply:
Seems by-design that dapps don't use stuff from `@agoric/internal`.

I often re-do them as 1-liners atop a file that I'm working in. I guess that happened more than once, so I factored them out.

Taking the suggestion to use `E` means this file is no longer a js-stdlib-only thing. hm. I wonder if that's a good move.

`mapValues` seems to be available as `objectMap` in `@endo/common`. I'd use that if it were already in my `node_modules`... but `yarn why` says it's not. `@endo/common` doesn't seem to be part of the agoric-upgrade-14 platform that I'm using.

I'm over-thinking this. I think I'll just leave it alone.